gis/grass/Portfile

```diff
@@ -1,11 +1,300 @@
+PortGroup           debug 1.0
 
+github.setup        OSGeo grass 8.0.0
 name                grass
-epoch               1
-version             6.4.6
-revision            5
-replaced_by         grass7
-PortGroup           obsolete 1.0
+set main_version    [join [lrange [split ${version} "."] 0 1] ""]
+revision            0
+set realVersion     ${version}
+distname            grass-${realVersion}
+maintainers         nomaintainer
+categories          gis
+license             GPL-2+
+platforms           darwin
 
+description         GRASS
+long_description    GRASS is a Geographic Information System (GIS) used for \
+                    geospatial data management and analysis.
+
+homepage            http://grass.osgeo.org/
+master_sites        ${homepage}grass[join [lrange [split ${realVersion} .] 0 1] {}]/source/
+
+checksums           rmd160  2192eb3ef991dd9645a7521f691aa9dc12bc4f16 \
+                    sha256  43231ae62874ace92b35d1457950749046aae374e4d9af9d9da22cf9c55d22a1 \
+                    size    64944539
+
+depends_build       port:pkgconfig
+depends_lib         port:bzip2 \
+                    port:freetype \
+                    port:fftw-3 \
+                    port:gdal \
+                    port:pdal \
+                    port:geos \
+                    port:tiff \
+                    port:libpng \
+                    port:liblas\
+                    path:lib/pkgconfig/cairo.pc:cairo \
+                    port:zlib \
+                    port:readline
+
+patchfiles          800-patch.diff
+
+configure.cflags    -DGL_SILENCE_DEPRECATION
+configure.ldflags   -Wl,-rpath,${prefix}/lib
+
+configure.args-append \
+    --prefix=${prefix}/share \
+    --exec-prefix=${prefix} \
+    --with-opengl=no \
+    --with-opencl=no \
+    --without-x \
+    --with-bzlib \
+    --with-bzlib-includes=${prefix}/include \
+    --with-bzlib-libs=${prefix}/lib \
+    --with-fftw-includes=${prefix}/include \
+    --with-fftw-libs=${prefix}/lib \
+    --with-freetype \
+    --with-freetype-includes=${prefix}/include/freetype2 \
+    --with-freetype-libs=${prefix}/lib \
+    --with-gdal=${prefix}/bin/gdal-config \
+    --with-pdal=${prefix}/bin/pdal-config \
+    --with-tiff-includes=${prefix}/include \
+    --with-tiff-libs=${prefix}/lib \
+    --with-png-includes=${prefix}/include \
+    --with-png-libs=${prefix}/lib \
+    --with-geos=${prefix}/bin/geos-config \
+    --with-liblas=${prefix}/bin/liblas-config \
+    --without-postgres \
+    --with-pthread \
+    --with-cairo \
+    --with-cairo-includes=${prefix}/include/cairo \
+    --with-cairo-libs=${prefix}/lib \
+    --with-blas=no \
+    --with-lapack=no \
+    --with-readline \
+    --with-zstd \
+    --with-zstd-includes=${prefix}/include \
+    --with-zstd-libs=${prefix}/lib \
+    --with-readline-includes=${prefix}/include/readline \
+    --with-readline-libs=${prefix}/lib
+
+build.target            default
+
+# Python 3 variants
+set python_suffixes {36 37 38 39}
```

Comment:
3.10 is the default version of python now. Does this software support it? If so, add 310 to python_suffixes here.

Reply:
GRASS v8 support Python 3.10, but wxPython 4.1.1. does not. So for now, no Python 3.10 for GRASS.